CMakeLists.txt

```diff
@@ -94,8 +94,8 @@ endif()
 
 # On Linux, use the lld linker, which is twice as fast for incremental debug builds
 if (NOT APPLE)
-    set(CMAKE_SHARED_LINKER_FLAGS "-fuse-ld=lld")
-    set(CMAKE_EXE_LINKER_FLAGS "-fuse-ld=lld")
+    set(CMAKE_SHARED_LINKER_FLAGS "-fuse-ld=lld-9")
+    set(CMAKE_EXE_LINKER_FLAGS "-fuse-ld=lld-9")
```

Comment:
This works for clang but not for g++: https://hyrise-ci.epic-hpi.de/blue/organizations/jenkins/hyrise%2Fhyrise/detail/PR-2169/19/pipeline/48/

@Bouncner did you verify that lld-9 works with g++? Otherwise should we #2167?

Reply:
Worked on 18.04 for me, but it fixed a clang issue and I relied on the CI. :-/
So no, haven't explicitly checked GCC.  :(

As it is unnecessary work to dive deep into this, just go back to `lld` and I'll take a look at the 18.04 LTS issue another time (still working ARM benchmarks).

Reply:
@mrks fine with reverting #2167?

Reply:
Sure

Reply:
One more thing to try is to explicitly install it using update-alternatives. I don't care if we do it here or separately.

Reply:
Separately!